From c8e2b1e0159342012eb60913185d4122bcdf3ad7 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 20 Feb 2021 02:24:33 -0500
Subject: [PATCH 003/181] Revert "FODCircleView: ensure that when we check for
 bouncer, we're on keyguard."

This reverts commit 9f34b9661810284300cd5a55ed5aff178ce06cc0.
---
 .../src/com/android/systemui/biometrics/FODCircleView.java      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java b/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java
index 4012e6112af..f32899fe3fb 100644
--- a/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java
+++ b/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java
@@ -381,7 +381,7 @@ public class FODCircleView extends ImageView {
             return;
         }
 
-        if (mIsKeyguard && mUpdateMonitor.getUserCanSkipBouncer(mUpdateMonitor.getCurrentUser())) {
+        if (mUpdateMonitor.getUserCanSkipBouncer(mUpdateMonitor.getCurrentUser())) {
             // Ignore show calls if user can skip bouncer
             return;
         }
-- 
2.17.1

